.github/workflows/cmake.yml

```diff
@@ -136,3 +136,51 @@ jobs:
     - name: Build Lagom with Fuzzers
       working-directory: ${{ github.workspace }}/Meta/Lagom/Build
       run: cmake --build .
+  build_and_test_on_macos:
+    runs-on: macos-11.0
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      # These packages are already part of the ubuntu-20.04 image:
+      # clang-format-10 cmake gcc-10 g++-10 shellcheck libgmp-dev
```

Comment:
But are they part of the macos-11.0 image?

Reply:
Whoops :sweat_smile:, good ol' copy and paste.